tools/autograd/templates/Functions.cpp

```diff
@@ -852,6 +852,23 @@ Tensor cholesky_backward(Tensor grad, bool upper, Tensor L) {
   return grad_input.add(grad_input.transpose(-1, -2)).mul_(0.5);  // Symmetrizing the gradient
 }
 
+Tensor cholesky_mod_backward(Tensor grad, bool upper, Tensor L, Tensor jitter) {
+  // L_tag = L - I * jiter[0], then use cholesky_backward
+  Tensor L_tag = L;
+  if (jitter.defined()) {
+    L_tag.copy_(L);
```

Comment:
```
Tensor  L_tag = L;
L_tag.copy_(L);
```
Could you please explain what this sequence is doing?

Reply:
removed this function